pages/components/LinkedButton.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { ComponentProps, MouseEventHandler, ReactPropTypes } from "react";
+
+export type ButtonTypes = {
+  buttonUrl: string;
+  content: string;
+  icon: any;
+  id: string;
+  onClick?: MouseEventHandler<HTMLAnchorElement>;
+};
+
+export const LinkedButton = ({
+  buttonUrl,
+  content,
+  icon,
+  id,
+  onClick,
+}: ButtonTypes) => {
+  return (
+    <>
+      <Link href={`/${buttonUrl}`} passHref>
+        <Button id={id} onClick={onClick}>
+          {content}
+          {icon}
+        </Button>
+      </Link>
+    </>
+  );
+};
+
+/*
+########
+STYLING
+########
+*/
+
+const Button = styled.a`
```

Comment:
Since this isn't really a button anymore you could consider giving this a more descriptive name.

Reply:
Alright, I'll call it ButtonLink, so that I still know it appears to be a button when seen in production, but in reality is just a styled link...